burstphoto/align/align.swift

```diff
@@ -18,7 +18,9 @@ func align_texture(_ ref_pyramid: [MTLTexture], _ comp_texture: MTLTexture, _ do
     alignment_descriptor.usage = [.shaderRead, .shaderWrite]
     alignment_descriptor.storageMode = .private
     var prev_alignment = device.makeTexture(descriptor: alignment_descriptor)!
+    
     var current_alignment = device.makeTexture(descriptor: alignment_descriptor)!
+    current_alignment.label = "\(comp_texture.label!.components(separatedBy: ":")[0]): Current alignment 0"
```

Comment:
Maybe use in label "Current alignment Start" as the number 0 will be added in the loop below.